Source/Widgets/FullscreenButton/FullscreenButtonViewModel.js

```diff
@@ -23,13 +25,23 @@ define([
      * @alias FullscreenButtonViewModel
      * @constructor
      *
+     * @param {Element|String} container The DOM element or ID that will contain the widget.
      * @param {Element|String} [fullscreenElement=document.body] The element or id to be placed into fullscreen mode.
      */
-    function FullscreenButtonViewModel(fullscreenElement) {
+    function FullscreenButtonViewModel(container, fullscreenElement) {
```

Comment:
This is a breaking change, at the very least we need to first deprecate the parameter to support backwards compatibility during deprecation.  Another option is to make the `container` parameter second and have it default to document.body, that would avoid the breaking change entirely and everything would continue to work as-is.